components/src/io_expander/pcf8574/pcf8574.adb

```diff
@@ -0,0 +1,51 @@
+--
+--  Copyright 2022 (C) Rolf Ebert
+--
+--  SPDX-License-Identifier: BSD-3-Clause
+--
+
+
+package body PCF8574 is
+
+   -----------------
+   --  Configure  --
+   -----------------
+
+   procedure Configure (This : in out PCF8574_Module;
+                        Port : Any_I2C_Port;
+                        Addr : PCF8574_Address)
+   is begin
+      This.Port := Port;
+      This.Addr := Addr;
+   end Configure;
+
+   -----------
+   --  Get  --
+   -----------
+
+   function Get (This : PCF8574_Module) return UInt8
+   is
+      Val    : I2C_Data (1 .. 1);
+      Status : I2C_Status;
+   begin
+      This.Port.Receive (This.Addr, Val, Status);
```

Comment:
`This.Port` is potentially `null`, you should check that before.
Same thing for the `Set` procedure.